tests/test_endpoints.py

```diff
@@ -107,5 +107,5 @@ def test_nhs_login_p9(nhsd_apim_proxy_url, nhsd_apim_auth_headers):
     }
     headers.update(nhsd_apim_auth_headers)
 
-    resp = requests.get(f"{nhsd_apim_proxy_url}/documents/Patient/9000000009", headers=headers)
+    resp = requests.get(f"{nhsd_apim_proxy_url}/FHIR/STU3/documents/Patient/9000000009", headers=headers)
```

Comment:
How about we make a env variable and populate its value to /FHIR/STU3/ and use that in all places

Reply:
yeah, good shout
d815d1b2b4936015aad76650a5639e1497a025c3